parsers/saarland.py

```diff
@@ -0,0 +1,223 @@
+            elif notice_short in SPECIAL_NOTICES:
+                notices_special.append(notice['displayName'])
+            else:
+                if sub_notices:
+                    notices_others.append(notice['displayName'])
+                else:
+                    notices_others.append(
+                        '%s %s' % (NOTICES_PREFIX_OTHERS, notice['displayName']))
+
+    if sub_notices:
+        notices_allergens = build_subnotices(notices_allergens, NOTICES_PREFIX_ALLERGENS)
+        notices_others = build_subnotices(notices_others, NOTICES_PREFIX_OTHERS)
+
+    return (notices_special + notices_allergens +
+            notices_others + notices_others_last)
+
+
+def build_subnotices(notices, prefix):
+    if len(notices) > 0:
+        notices[0] = '%s %s' % (prefix, notices[0])
+    if len(notices) > 1:
+        notices[-2] = (
+            '%s und %s' % (notices[-2], notices[-1]))
+        del notices[-1]
+    return notices
+
+
+def build_notes(notices, components):
+    components_all = []
+
+    notices_list = get_notices(notices)
+
+    for component in components:
+        component_string = '%s %s' % (NOTICES_PREFIX_COMPLEMENT, component['name'])
+        if len(component['notices']) > 0:
+            component_notices_list = get_notices(component['notices'], True)
+            component_string += ' (%s)' % ', '.join(component_notices_list)
+        components_all.append(component_string)
+
+    return components_all + notices_list
+
+
+def build_hours(opening_hours):
+    if opening_hours is None:
+        return []
+
+    start = datetime.datetime.strptime(
+        opening_hours['start'], UTC_DATE_STRING).time()
+    end = datetime.datetime.strptime(
+        opening_hours['end'], UTC_DATE_STRING).time()
+    return [
+        'verfügbar von %s bis %s Uhr' %
+        (datetime.time.strftime(start, "%H:%M"),
+            datetime.time.strftime(end, "%H:%M"))
+    ]
+
+
+def build_location(description):
+    return ['Ort: %s' % description] if description else []
+
+
+def parse_url(url, today=False):
+    global base_data
+
+    load_base_data()
+
+    canteen = LazyBuilder()
+    with urlopen(url) as response:
+        data = json.loads(response.read().decode())
+
+    for day in data['days']:
+        date = datetime.datetime.strptime(day['date'], UTC_DATE_STRING).date()
+
+        if today and (datetime.date.today() != date):
+            continue
+
+        for counter in day['counters']:
+            counter_name = counter['displayName']
+            counter_description = counter['description']
+            counter_hours = counter.get('openingHours')
+
+            for meal in counter['meals']:
+                if 'knownMealId' in meal:
+                    # This is meant to allow recognizing recurring meals,
+                    # for features like marking meals as favorites.
+                    # Up to now, not really used in the mensaar.de API.
+                    print('knownMealId: %s' % meal['knownMealId'])
```

Comment:
I's fine if you want to keep it, but please print to stderr with `file=sys.stderr`.

Reply:
Okay, I've changed it.